src/shaders/ptex-default-gl410.frag

```diff
@@ -29,7 +29,7 @@ const uint FACE_MASK = 0x3FFFFFFF;
 
 int GetAdjFace(int face, int edge, out int rot) {
   // uint data = meshAdjFaces[face * 4 + edge];
-  uint data = uint(texelFetch(meshAdjFaces, face * 4 + edge));
+  uint data = texelFetch(meshAdjFaces, face * 4 + edge).r;
```

Comment:
@mosra : Is it the correct way to fetch the unsigned int in the buffer texture? 
BTW, please note: I changed the meshAdjFaces from sampleBuffer to usamplerBuffer.

Reply:
If the shader compiler doesn't complain, yes :)

Reply:
The previous code will not be complained by the compiler either. But I doubt if that is correct. So I changed it.